Sources/Ruby/RubyMockServer.swift

```diff
@@ -46,25 +46,38 @@ open class RubyMockServer: MockServer {
 
     // MARK: URLRequestConvertible
     func asURLRequest() throws -> URLRequest {
-      let url = try Router.baseURLString.asURL()
+      guard let url = URL(string: Router.baseURLString) else { throw NSError(domain: "", code: 1, userInfo: nil) }
```

Comment:
A new merge on the origin:master made Router method to `throw`. This is kind of out of date now. :|

Reply:
TODO:
Yeah, I'll have to see about that